### ISSUE TITLE
docblock pragmas doesn't exists in allure results

### ISSUE THREAD

**Issue by KillWolfVlad** [c4lifa/jest-allure-circus#43]
## Describe the bug

docblock pragmas doesn't exists in allure results.

## To Reproduce

Create simple test:

```typescript
describe("Simple Test", () => {
  it("must parse docblock", () => {
    /** My test description.
     * @epic Implement addition functionality
     * @tag Accounting
     */

    expect(1).toBe(1);
  });
});
```

## Expected behavior

Test in allure results have `epic` and `tag`.

## Screenshots

Actual result:

![image](https://user-images.githubusercontent.com/18448635/182531202-af930ca4-7ca6-4be2-8373-a52d137c9618.png)

Expected result:

![image](https://user-images.githubusercontent.com/18448635/182531353-e5a610a7-9fbc-4c3d-9b80-c286d1403859.png)

## Additional context

I have this jest packages:
* "jest": "28.1.3"
* "jest-allure-circus": "1.0.15"
* "jest-extended": "3.0.1"
* "ts-jest": "28.0.7"

**Comment by KillWolfVlad** [comment]
My fast patch to fix this issue, maybe we also need remove `@ts-expect-error`?

```patch
--- a/src/allure-reporter.ts
+++ b/src/allure-reporter.ts
@@ -346,7 +346,7 @@ export default class AllureReporter {
 
 	private setAllureLabelsAndLinks(currentTest: AllureTest, labelName: string, value: string, index?: number) {
 		// @ts-expect-error (ts(2341))
-		if (currentTest.info.name === this.testNames[index]) {
+		if (index === undefined || currentTest.info.name === this.testNames[index]) {
 			switch (labelName) {
 				case 'issue':
 					currentTest.addLink(`${this.jiraUrl}${value}`, value, LinkType.ISSUE);
```

**Comment by c4lifa** [comment]
Thanks.

- Fix added to 1.0.16beta.
